.editorconfig

```
@@ -0,0 +1,11 @@
+root = true
+[*]
+indent_size = 4
+indent_style = space
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = false
```

```suggestion
insert_final_newline = true
```
Seria interessante ter esse valor como true, isso é mais cosmético, porém é uma configuração comum em vários projetos, o plugin [Editor config](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) instalado no seu vscode, para adicionar uma linha no final do arquivo evitar o que aconteceu no arquivo abaixo.